spyvm/test/test_external_calls.py

```diff
@@ -123,12 +124,27 @@ def test_fileplugin_filewrite_words(monkeypatch):
     def write(fd, data):
         assert len(data) == 4
         assert data == 'dcba'
+        return 4
     monkeypatch.setattr(os, "write", write)
 
     content = model.W_WordsObject(space, space.w_String, 1)
     content.words = [rffi.r_uint(1633837924)]
     try:
-        stack = [space.w(1), space.w(1), content, space.w(0), space.w(1)]
+        stack = [space.w(1), space.w(1), content, space.w(1), space.w(1)]
+        w_c = external_call('FilePlugin', 'primitiveFileWrite', stack)
+    finally:
+        monkeypatch.undo()
+
+def test_fileplugin_filewrite_float(monkeypatch):
+    def write(fd, data):
+        assert len(data) == 4
+        assert data == 'dcba'
+        return 4
+    monkeypatch.setattr(os, "write", write)
+
+    content = space.wrap_float(2.6100787562286154e+20)
+    try:
+        stack = [space.w(1), space.w(1), content, space.w(1), space.w(1)]
         w_c = external_call('FilePlugin', 'primitiveFileWrite', stack)
     finally:
         monkeypatch.undo()
```

Comment:
It has already been like this before the PR, but I would like to point out here, that monkeypatching does not need to be undone. The respective py.test plugin takes care of the teardown.